fixtures/resource/update_resource_fixtures.py

```diff
@@ -0,0 +1,76 @@
+
+null = None
+
+update_room_resource_query = '''
+                        mutation{
+                        updateRoomResource(roomId:1,resourceId:1,name:"TV Screen"){
```

Comment:
Adjust this to reflect that there is no need to query with roomID, resource ids are enough, room id should just be a field in the kwargs